packages/landing-gatsby/src/templates/processTamplate.js

```diff
@@ -0,0 +1,342 @@
+import React, { useState, Fragment } from 'react';
+import { ThemeProvider } from 'styled-components';
+import { Modal } from '@redq/reuse-modal';
+import { appTheme } from 'common/src/theme/app';
+import { GlobalStyle, AppWrapper, Sun } from '../containers/App/app.style';
+import {
+  Tabs,
+  Tab,
+  Content,
```

Comment:
/packages/landing-gatsby/src/templates/processTamplate.js  9:3  warning  'Content' is defined but never used  no-unused-vars